docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -291,11 +291,20 @@ paths:
           minimum: 0
           maximum: 200
           default: 20
+        - name: search_after
+          in: query
+          description: Returns results after the annotation who's sort field has this value. 
+            If specifying a date use the format yyyy-MM-dd'T'HH:mm:ss.SSX or time in 
+            miliseconds since the epoch. This is used for iteration through large collections 
```

Comment:
In testing it appears that the full date is mandatory but each time component is optional. This seems like useful behaviour. Might be worth noting.

Reply:
The full date isn't mandatory but this was an excellent catch-thank you!! What was happening was the year ("2017") was successfully getting cast as a float and interpreted as ms since the epoch instead of going through the date parser. I added some logic to address this issue. Let me know if it makes sense!

Reply:
Ah yes, I see. I found an issue that if you specify only the year, `dateutil.parser.parse` assumes the _current_ month and day rather than the first month and day of the year, which is what I would have expected.